client/src/components/GoogleSignIn.jsx

```
@@ -0,0 +1,59 @@
+      console.error('Google login error:', error);
+      const errorMessage =
+        error.response?.data?.message || 'Google login failed. Try again.';
+      toast.error(errorMessage);
+    }
+  };
+
+  return (
+    <button
+      onClick={handleGoogle}
+      className="cursor-pointer flex items-center justify-center w-full bg-white border border-gray-300 text-gray-700 p-2 rounded-md shadow-md hover:bg-gray-100"
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)